docs/presentations/2024-08-27-CU-Anschutz-DBMI-Retreat/_extensions/quarto-ext/poster/typst-show.typ

```diff
@@ -0,0 +1,74 @@
+// Typst custom formats typically consist of a 'typst-template.typ' (which is
+// the source code for a typst template) and a 'typst-show.typ' which calls the
+// template's function (forwarding Pandoc metadata values as required)
+//
+// This is an example 'typst-show.typ' file (based on the default template
+// that ships with Quarto). It calls the typst function named 'article' which
+// is defined in the 'typst-template.typ' file.
+//
+// If you are creating or packaging a custom typst template you will likely
+// want to replace this file and 'typst-template.typ' entirely. You can find
+// documentation on creating typst templates here and some examples here:
+//   - https://typst.app/docs/tutorial/making-a-template/
+//   - https://github.com/typst/templates
+
+#show: doc => poster(
+  $if(title)$ title: [$title$], $endif$
+  // TODO: use Quarto's normalized metadata.
```

Comment:
Is this related to an us TODO or is this from a template?

Reply:
Thanks good question, this is from the template.